docs/misp2_installation_manual_18.04.md

```diff
@@ -694,6 +696,39 @@ sudo -i
     service apache2 restart
     ```
 
+#### 5.7.1 Additional ID-card configuration options
+
+It should be noted that the default configuration is set up so that the authentication certificate is requested on the
+root of the application. This means that the session does not get renegotiated until the user has closed the browser
```

Comment:
> This means that the session does not get renegotiated until the user has closed the browser tab

Maybe "terminated" would be more descriptive instead of "renegotiated"?

Reply:
Fixed.